### PR TITLE
refactoring code with Assign Multiple Targets

### PR DESCRIPTION
**Pull request by idiomaticrefactoring**
refactoring code with Assign Multiple Targets which is more pythonic, concise, readable and efficient;how do think this change which has practical value？

### CODE REVIEW

**Comment by lmcinnes** [comment]
It a perfectly fine change. That code is not actually executed by anything at the moment, so its not exactly a high priority. Any particular reason for tackling this item?

File: pynndescent/threaded_rp_trees.py [hunk 1]
```diff
@@ -73,11 +73,8 @@ def make_angular_hyperplane(data, indices, rng_state):
     right_index = tau_rand_int(rng_state) % indices.shape[0]
     right_index += left_index == right_index
     right_index = right_index % indices.shape[0]
-    left = indices[left_index]
-    right = indices[right_index]
-
-    left_norm = norm(data[left])
-    right_norm = norm(data[right])
+    left , right = indices[left_index], indices[right_index]
+    left_norm , right_norm  = norm(data[left]), norm(data[right])
```
Comment (jamestwebber) [inline pynndescent/threaded_rp_trees.py:77]:
```suggestion
    left, right = indices[left_index], indices[right_index]
    left_norm, right_norm  = norm(data[left]), norm(data[right])
```

**Comment by alik604** [comment]
I think it's a bot, definitely not a _hooman_. [Another pull by them](https://github.com/online-ml/river/pull/842). I like the original more because I find it more readable.

**Comment by jamestwebber** [comment]
Gonna close this. The PR itself is pointless and the originating account seems to be a bot now.

**Pull request closed without merging**